frontend/src/static/js/v_authorship.js

```diff
@@ -257,7 +259,6 @@ window.vAuthorship = {
 
       Object.keys(this.filesLinesObj).forEach((file) => {
         if (this.filesLinesObj[file] !== 0) {
-          this.selectedFileTypes.push(file);
```

Comment:
redundant call, could this in a separate pr if preferred.
( there is a `this.selectedFileTypes = this.fileTypes.slice();` right below it )